plugins/rapid7_insightappsec/help.md

```diff
@@ -13,6 +13,10 @@ This plugin utilizes the [Rapid7 InsightAppSec API](https://insightappsec.help.r
 
 * Requires an API Key from Insight platform
 
+# Supported Product Versions
+
+* Rapid7 InsightAppSec
+
```

Comment:
Add release date of tested version:
https://docs.rapid7.com/release-notes/insightappsec/20220228/

Reply:
@mgrandys-r7  The changes were cosmetic. Plugin was not tested. I don't think I can write that.

Reply:
If there is no timestamp to report testing the plugin, please simply leave the `supported_versions` field empty and regenerate the plugin. Thank you.